src/update/reward_history.py

```diff
@@ -59,7 +61,7 @@ def __str__(self) -> str:
             map(pg_hex2bytea, [self.solver, self.tx_hash, self.order_uid])
         )
         safe = self.safe_liquidity if self.safe_liquidity is not None else "Null"
-        return f"('{order_id}','{solver}','{tx_hash}',{self.amount},{safe})"
+        return f"('{order_id}','{solver}','{tx_hash}', {self.surplus_fee},{self.amount},{safe})"
```

Comment:
nit: Looks like a superfluous ` ` space.
```suggestion
        return f"('{order_id}','{solver}','{tx_hash}',{self.surplus_fee},{self.amount},{safe})"
```

Reply:
I will add this fix in the follow up PR